src/tests/FunctionalTests/Android/Device_Emulator/gRPC/grpc-dotnet/testassets/InteropTestsWebsite/InteropTestsWebsite.csproj

```diff
@@ -21,9 +21,13 @@
     <None Include="..\Certs\InteropTests\*.*" LinkBase="Certs">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+  </ItemGroup>
+  
+  <Import Project="Versions.props" />
```

Comment:
this looks wrong, there's no Versions.props in the folder

Reply:
the build happens inside of a docker container, so we need to copy the file during the build and I forgot to commit the change to the csproj file, should be fixed now